### PR TITLE
[4819][ADD] product_view_adjust

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4819](https://www.quartile.co/web#id=4819&cids=3&model=project.task&view_type=form)

### CODE REVIEW

**Comment by kanda999** [comment]
@AungKoKoLin1997 

Could you please check the contents of the product_ext_sst module once?
Would it be better to have both modules?

**Comment by AungKoKoLin1997** [comment]
> @AungKoKoLin1997
> 
> Could you please check the contents of the product_ext_sst module once? Would it be better to have both modules?

@kanda999  Actually, I checked before I create new module. I create new module because `product_ext_sst` depends on the custom modules and make adjustments. So, if some of these custom modules are uninstalled, we may have to adjust(may be corner case). That's why I believe new module with only `delivery` dependency is better choice and can add more adjustments related to odoo standard codes. If we want to make adjustment from custom module, we should work on it's own module.

**Comment by kanda999** [comment]
@AungKoKoLin1997 
Thank you for your input, then I will check this module.

File: product_view_adjust/views/product_template_views.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <field name="arch" type="xml">
+            <xpath
+                expr="//group[@name='group_lots_and_weight']/field[@name='hs_code']"
+                position="attributes"
+            >
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//field[@name='categ_id']" position="after">
+                <field name="hs_code" />
+            </xpath>
```
Comment (yostashiro) [inline product_view_adjust/views/product_template_views.xml]: How about this?
```suggestion
            <xpath expr="//field[@name='categ_id']" position="after">
                <xpath expr="//field[@name='hs_code']" position="move" />
            </xpath>
```

**Review by yostashiro — APPROVED** [review]

**Review by kanda999 — APPROVED** [review]
Functinal check

**Pull request merged**